js/toggle-menu.js

```diff
@@ -0,0 +1,95 @@
+function createMyElement(el, ids = '', cl = '') {
+  const doc = document.createElement(el);
+
+  if (ids !== '') {
+    doc.id = ids;
+  }
+
+  if (cl !== '') {
+    doc.className = cl;
+  }
+  return doc;
+}
+
+function createMyMenu() {
+  const container = createMyElement('div', 'nav-toggle');
+
+  // create button
+  const toggleBtnContainer = createMyElement('div', 'close-button-container');
+  const toggleBtn = createMyElement('img', 'img-close');
+  toggleBtn.setAttribute('src', './../images/x.svg');
+  toggleBtnContainer.appendChild(toggleBtn);
+  container.appendChild(toggleBtnContainer);
```

Comment:
- [ ] [OPTIONAL] Good job so far! Please fix the error of `X` **icon** in the picture because it is not showing for me maybe it is my mistake. 
- Please, if you have this icon, leave a screenshot of the menu `X` **icon** under this suggestion.

![image](https://user-images.githubusercontent.com/63915024/181036695-6540f583-69cd-4bea-a46e-37d7cc8219eb.png)

Reply:
Thank you very much [Muhammad Rizo](https://github.com/mukhammadrizooff). 
You are right. It works fine here on my laptop and phone, but I think for it to work generally, I should count the path from the root, instead of the js folder. Now fixed. thank you